graph/cytoscape/cytoscape.go

```diff
@@ -189,19 +189,20 @@ func addRate(ed *EdgeData, sn *tree.ServiceNode, nd *NodeData, o options.VendorO
 		rateErr := rate3xx + rate4xx + rate5xx
 		percentErr := rateErr / rate * 100.0
 
-		ed.Rate = fmt.Sprintf("%.2f", rate)
+		ed.Rate = fmt.Sprintf("%.4f", rate)
 		nd.Rate = add(nd.Rate, rate)
 		if rate3xx > 0.0 {
-			ed.Rate3xx = fmt.Sprintf("%.2f", rate3xx)
+			ed.Rate3xx = fmt.Sprintf("%.4f", rate3xx)
 			nd.Rate3xx = add(nd.Rate3xx, rate3xx)
 		}
 		if rate4xx > 0.0 {
-			ed.Rate4xx = fmt.Sprintf("%.2f", rate4xx)
+			ed.Rate4xx = fmt.Sprintf("%.4f", rate4xx)
 			nd.Rate4xx = add(nd.Rate4xx, rate4xx)
 		}
 		if rate5xx > 0.0 {
-			ed.Rate5xx = fmt.Sprintf("%.2f", rate5xx)
+			ed.Rate5xx = fmt.Sprintf("%.4f", rate5xx)
 			nd.Rate5xx = add(nd.Rate5xx, rate5xx)
+			log.Infof("RATE rate5xx=%v, edge5xx=%v, node5xx=%v", rate5xx, ed.Rate5xx, nd.Rate5xx)
```

Comment:
do we really want this logged at info level? seems like a debugging message, should be at DEBUG level.